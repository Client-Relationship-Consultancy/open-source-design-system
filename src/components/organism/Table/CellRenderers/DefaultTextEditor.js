/**
 * This may seem pointless but the very act of using a renderer adds a span to the cell
 * and applied the css to that span in table.js. Its the only way i could get it working.
 */
export function DefaultTextEditor (props) {
  return props.value;
}

export default DefaultTextEditor
