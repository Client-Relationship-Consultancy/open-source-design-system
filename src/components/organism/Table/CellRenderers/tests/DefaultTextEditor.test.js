import React from "react"
import { shallow } from "enzyme"
import "jest-styled-components"
import DefaultTextEditor from "../DefaultTextEditor"

const component = shallow(<DefaultTextEditor value="This is inside the div" />)

describe("DefaultTextEditor test", () => {
  it("should match last snapshot", () => {
    expect(component).toMatchSnapshot()
  })
})
