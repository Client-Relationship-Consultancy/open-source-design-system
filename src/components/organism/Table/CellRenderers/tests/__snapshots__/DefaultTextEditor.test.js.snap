// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultTextEditor test should match last snapshot 1`] = `
<DefaultTextEditor__Cell
  className="DefaultTextEditor"
>
  This is inside the div
</DefaultTextEditor__Cell>
`;
